docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -190,8 +190,21 @@
               (node_nf_conntrack_entries / node_nf_conntrack_entries_limit) > 0.75
             ||| % $._config,
             annotations: {
-              summary: 'Number of conntrack are getting close to the limit',
-              description: '{{ $value | humanizePercentage }} of conntrack entries are used',
+              summary: 'Number of conntrack are getting close to the limit.',
+              description: '{{ $value | humanizePercentage }} of conntrack entries are used.',
+            },
+            labels: {
+              severity: 'warning',
+            },
+          },
+          {
+            alert: 'NodeTextFileCollectorScrapeError',
+            expr: |||
+              node_textfile_scrape_error{%(nodeExporterSelector)s} == 1
```

Comment:
umm ... why is this a gauge?

Reply:
oh I get it, it's a synchronous part of the collector

Reply:
kind of an odd pattern but until we change this this alert seems ok

Reply:
Well it's a state variable, like 'failed'. Actually maybe it should be called node_textfile_failed or something to avoid confusion with error counters.